Studio-Lovelies/GG-JointJustice-Unity
# Apple Silicon compatibility

## Summary
This PR contains various changes to make development on Apple Silicon possible (and easier):
- Replaced instance of `Moq` inside `/Plugins`-folder with reference inside `package.json` (which is then loaded via the Unity Package Manager on-demand)
- Added global `.gitignore` to filter `.DS_Store` <sup>which is still a thing in 2022; macOS plz</sup>
- Removed GitHub for Unity plugin [due to deprecation](https://assetstore.unity.com/packages/tools/version-control/github-for-unity-118069)

## Testing instructions
1. Verify that literally nothing has changed from a player's perspective
2. Verify that you can still open the Unity Editor without encountering errors and make changes to the game
